spec/lib/insights/api/common/rbac/access_spec.rb

```diff
@@ -34,7 +34,7 @@
   end
 
   it "rbac is enabled by default" do
```

Comment:
This test should have a better description like "rbac is disabled with ENV var BYPASS_RBAC=true"